test/dotnet-pack.Tests/PackTests.cs

```diff
@@ -95,6 +95,29 @@ public void HasBuildOutputWhenUsingBuildBasePath()
             zip.Entries.Should().Contain(e => e.FullName == "lib/netstandardapp1.5/TestLibraryWithConfiguration.dll");
         }
 
+        [Fact]
+        public void PackAddsCorrectFilesForProjectsWithOutputNameSpecified()
```

Comment:
If you feel strongly about this I can add a test like this but I think this is implicitly tested almost everywhere because we only have less than a handful of test projects that have outputName specified. As I pointed above the `OutputName` property falls back to `Project.Name`.